### PR TITLE
Commerce 1453 api

### PR DESCRIPTION
**Pull request by alecsloan**
Fixes all JavaAPISignatureCheck suppressions for commerce-api module except CommerceContextFactory which is dependent on other fixes.

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Review by ethanbustad — APPROVED** [review]

File: commerce-api/src/main/java/com/liferay/commerce/starter/CommerceRegionsStarter.java [hunk 1]
```diff
@@ -16,14 +16,12 @@
...
-import com.liferay.portal.kernel.service.ServiceContext;
-
 /**
  * @author Marco Leo
  */
 @ProviderType
 public interface CommerceRegionsStarter {
 
-	public void start(ServiceContext serviceContext) throws Exception;
+	public void start(long companyId, long userId) throws Exception;
```
Comment (marco-leo) [inline commerce-api/src/main/java/com/liferay/commerce/starter/CommerceRegionsStarter.java:25]: if you have user you don't need company

File: commerce-payment-service/src/main/java/com/liferay/commerce/payment/internal/util/CommercePaymentUtilsImpl.java [hunk 2]
```diff
@@ -141,14 +136,12 @@ public CommercePaymentRequestProvider getCommercePaymentRequestProvider(
...
-		HttpServletRequest httpServletRequest, CommerceOrder commerceOrder,
-		String redirect, CommercePaymentMethod commercePaymentMethod,
-		int extraCapacity) {
+		CommerceOrder commerceOrder, String redirect,
+		CommercePaymentMethod commercePaymentMethod, int extraCapacity) {
 
 		StringBundler sb = new StringBundler(
 			extraCapacity + (Validator.isNotNull(redirect) ? 13 : 11));
 
-		sb.append(_portal.getPortalURL(httpServletRequest));
```
Comment (marco-leo) [inline commerce-payment-service/src/main/java/com/liferay/commerce/payment/internal/util/CommercePaymentUtilsImpl.java:151]: this is needed

**Comment by marco-leo** [comment]
We can't remove those suppressions

**Pull request closed without merging**